### PR TITLE
style(tests): format many tests with black

### PR DESCRIPTION
**Pull request by dshemetov**
Part of #962 

### Summary:

- Runs the tests and a few `common` dir files through black
- Holds off on `covid_hosp` and `covidcast` until the PRs on those are merged

### Prerequisites:

- [x] Unless it is a documentation hotfix it should be merged against the `dev` branch
- [x] Branch is up-to-date with the branch to be merged with, i.e. `dev`
- [x] Build is successful
- [x] Code is cleaned up and formatted

### CODE REVIEW

File: src/common/covidcast_row.py [hunk 1]
```diff
@@ -95,9 +102,13 @@ def as_api_row_df(self, ignore_fields: Optional[List[str]] = None) -> pd.DataFra
         """Returns a dataframe view into the row with the fields returned by the API server."""
         return self.as_dataframe(ignore_fields=self._api_row_ignore_fields + (ignore_fields or []))
 
-    def as_api_compatibility_row_df(self, ignore_fields: Optional[List[str]] = None) -> pd.DataFrame:
+    def as_api_compatibility_row_df(
+        self, ignore_fields: Optional[List[str]] = None
+    ) -> pd.DataFrame:
```
Comment (krivard) [inline src/common/covidcast_row.py:107]: multi-line function signatures are not my favorite but i will hold my nose

File: tests/acquisition/fluview/test_impute_missing_values.py [hunk 2]
```diff
@@ -7,125 +7,129 @@
...
 # first party
 from delphi.utils.geo.locations import Locations
-from delphi.epidata.acquisition.fluview.impute_missing_values import get_argument_parser, \
-  get_lag_and_ili, impute_missing_values, StatespaceException
+from delphi.epidata.acquisition.fluview.impute_missing_values import (
+    get_argument_parser,
+    get_lag_and_ili,
+    impute_missing_values,
+    StatespaceException,
+)
```
Comment (krivard) [inline tests/acquisition/fluview/test_impute_missing_values.py:15]: 🙌

File: tests/common/test_covidcast_row.py [hunk 3]
```diff
@@ -7,49 +7,47 @@
...
-        ]
+        ),
+        CovidcastRow._pandas_dtypes,
+    )
 
+    def test_transpose_dict(self):
+        assert transpose_dict({"a": [2, 4, 6], "b": [3, 5, 7], "c": [10, 20, 30]}) == [
+            {"a": 2, "b": 3, "c": 10},
+            {"a": 4, "b": 5, "c": 20},
+            {"a": 6, "b": 7, "c": 30},
```
Comment (krivard) [inline tests/common/test_covidcast_row.py:49]: nope nope, whitespace is always semantic for matrix operations

File: tests/common/test_covidcast_row.py [hunk 4]
```diff
@@ -7,49 +7,47 @@
...
+                "direction": [None] * 10,
             }
-        ) == [
-            {"a": 2, "b": 3, "c": 10}, 
-            {"a": 4, "b": 5, "c": 20}, 
-            {"a": 6, "b": 7, "c": 30}
-        ]
+        ),
+        CovidcastRow._pandas_dtypes,
+    )
```
Comment (krivard) [inline tests/common/test_covidcast_row.py:43]: this feels ridiculous but i don't think it's worth manually reformatting?

File: src/common/covidcast_row.py [hunk 5]
```diff
@@ -62,7 +63,11 @@ class CovidcastRow:
     # Classvars.
     _db_row_ignore_fields: ClassVar = []
     _api_row_ignore_fields: ClassVar = ["epimetric_id", "value_updated_timestamp"]
-    _api_row_compatibility_ignore_fields: ClassVar = _api_row_ignore_fields + ["source", "time_type", "geo_type"]
+    _api_row_compatibility_ignore_fields: ClassVar = _api_row_ignore_fields + [
+        "source",
+        "time_type",
+        "geo_type",
+    ]
```
Comment (krivard) [inline src/common/covidcast_row.py:70]: 🙌

**Review by krivard — COMMENTED** [review]
largely good, one item to revert

File: tests/common/test_covidcast_row.py (hunk 4 above)
Comment (dshemetov) [reply tests/common/test_covidcast_row.py:43]: yea, that's kinda where i am with it. it doesn't look great, but at least it will be consistent?

**Review by krivard — APPROVED** [review]
👍

**Pull request merged**